panamaconverter/src/PanamaCIRCTConverter.scala

```diff
@@ -883,6 +883,21 @@ class PanamaCIRCTConverter {
   def passManager(): PanamaCIRCTPassManager = new PanamaCIRCTPassManager(circt, mlirRootModule)
   def om():          PanamaCIRCTOM = new PanamaCIRCTOM(circt, mlirRootModule)
 
+  def foreachHwModule(callback: String => Unit) = {
```

Comment:
I personally don't like this API...
But... let's make it work and design the entire workflow in the future.